### ISSUE TITLE
npm audit is reporting a potential security risk on the module; `request` dependency does not allow minor version bumps

### ISSUE THREAD

**Issue by boutell** [Azure/azure-storage-node#449]
We are one vulnerability away from 100% npm audit compliance in our project... it is due to the minor version of `request` that `azure-storage-node` is locked down to (2.83.x). In the current release of `request` (2.86.x) there is no reported vulnerability.

As you can see the vulnerability is in the `stringstream` module used by the `request` module version you have in your `package.json`.

Please change your dependency to `^2.86.0` so that it automatically follows both patchlevel and minor version changes, as semver indicates this should not give you any breaking changes, or if you prefer to validate minor version changes by hand as needed, bump it at least to `~2.86.0`.

With these changes `azure-storage-node` should not throw any `npm audit` warnings.

Thanks!

```
                       === npm audit security report ===                        
                                                                                
┌──────────────────────────────────────────────────────────────────────────────┐
│                                Manual Review                                 │
│            Some vulnerabilities require your attention to resolve            │
│                                                                              │
│         Visit https://go.npm.me/audit-guide for additional guidance          │
└──────────────────────────────────────────────────────────────────────────────┘
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ Moderate      │ Out-of-bounds Read                                           │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ stringstream                                                 │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ No patch available                                           │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ uploadfs                                                     │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ uploadfs > azure-storage > request > stringstream            │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://nodesecurity.io/advisories/664                       │
└───────────────┴──────────────────────────────────────────────────────────────┘

[!] 1 vulnerability found - Packages audited: 2193 (728 dev, 154 optional)
    Severity: 1 Moderate
```

**Comment by XiaoningLiu** [comment]
@boutell Thanks for reporting! We'll update the version.

**Comment by XiaoningLiu** [comment]
Released 2.8.3 for this issue.
